packages/conf-libclang/conf-libclang.15/files/configure.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash -ex
```

Comment:
What's the difference between conf-libclang.14 and conf-libclang.15 ?

Reply:
They differ in `files/configure.sh`: `conf-libclang.14` searches for LLVM libraries up to version 14 (included), whereas `conf-libclang.15` searches for LLVM libraries up to version 15 (included). The purpose is to be able to express that `clangml.4.7.0` for instance requires `conf-libclang<15` (because the bindings only support clang 14 and below) and `clangml.4.8.0` requires `conf-libclang<16` (because the bindings only support clang 15 and below).

Reply:
Full diff:
```diff
diff --color -r conf-libclang.14/files/configure.sh conf-libclang.15/files/configure.sh
9c9
< maximum_version=14
---
> maximum_version=15
diff --color -r conf-libclang.14/opam conf-libclang.15/opam
32c32
<   "configure.sh" "sha512=901562da7c7d5016b2b348ce4bf05ea2173b78e66d64f962a5cef0686b8df33c850ee26093ca8c00163043fc2506681fc014681ff75f7fcfdc5a57a67a48e162"
---
>   "configure.sh" "sha512=495b1915a845c75033449e8a86b35e399320a50d63cee959f5685dedb085c5efc2a27d5c2672c90c38444a5af4edd212cf2e23bb2b34aaedc5cf271073dce087"
34c34
< synopsis: "Virtual package relying on the installation of llvm and clang libraries (<= 14.0.x)"
---
> synopsis: "Virtual package relying on the installation of llvm and clang libraries (<= 15.0.x)"
```